test/Domain/LeanCode.DomainModels.MassTransitRelay.Tests/ScopedFiltersTests.cs

```diff
@@ -10,34 +10,44 @@
 
 namespace LeanCode.DomainModels.MassTransitRelay.Tests;
 
-public class ScopedFiltersTests : IAsyncLifetime
+public class ScopedFiltersTests : IAsyncLifetime, IDisposable
```

Comment:
`IAsyncDisposable` as well?

Reply:
xUnit does not use `IAsyncDisposable` at all.